packages/docs/src/components/content/StorybookExampleFooter.tsx

```diff
@@ -18,104 +16,21 @@ interface StorybookExampleFooterProps {
-  );
-
-  const onIframeLoad = () => {
-    const errorLoadingReactCode = () => setReactCode('Error loading React source');
-    const errorLoadingHtmlCode = () => setHtmlCode('Error loading HTML source');
-
-    if (!iframeRef.current) {
-      errorLoadingReactCode();
-      errorLoadingHtmlCode();
-      return;
-    }
-
-    const normalizedStoryId = storyId.split('&')[0]; // Omit additional args
-    const storyBlockSelector = `#anchor--${normalizedStoryId}`;
-    const storyRootSelector = `#story--${normalizedStoryId}`;
-    const codeButtonSelector = `${storyBlockSelector} .docblock-code-toggle`;
-    const codeBlockSelector = `${storyBlockSelector} .language-jsx`;
-    const body = iframeRef.current.contentDocument.body;
-
-    const storyRootEl = body.querySelector(storyRootSelector);
-    if (storyRootEl) {
-      setHtmlCode(storyRootEl.innerHTML);
-    } else {
-      errorLoadingHtmlCode();
-    }
-
-    // Find the 'Show code' button and click it so we can access the code
-    const showCodeButton = body.querySelector(codeButtonSelector);
-    if (!(showCodeButton && (showCodeButton as HTMLButtonElement).click)) {
-      console.error(
-        `Code button missing or invalid using this selector: '${codeButtonSelector}'`,
-        showCodeButton
-      );
-      errorLoadingReactCode();
-      return;
-    }
-    // Hide the iframe first so that clicking the button doesn't scroll us to it. We can't
-    // do this when we first render the iframe or it will never load.
-    iframeRef.current.hidden = true;
-    iframeRef.current.style.display = 'none';
-    (showCodeButton as HTMLButtonElement).click();
-
-    // Read the code out of the resulting code block after waiting for it to be generated
-    let retries = 0;
-    const MAX_RETRIES = 3;
-    function readCode() {
-      setTimeout(() => {
-        const codeEl = body.querySelector(codeBlockSelector);
-        if (codeEl) {
-          setReactCode(codeEl.innerHTML);
-        } else if (retries < MAX_RETRIES) {
-          retries++;
-          readCode();
-        } else {
-          console.error(`Code block not found: ${codeBlockSelector}`, codeEl);
-          errorLoadingReactCode();
-        }
-      }, 1000);
-    }
-    readCode();
-  };
-
-  const iframe = (
-    <iframe
-      className="c-storybook-example-footer__iframe"
-      referrerPolicy="no-referrer"
-      src={iframeUrl}
-      ref={iframeRef}
-      loading="lazy"
-      onLoad={onIframeLoad}
-      tabIndex={-1}
-    />
-  );
-
   return (
     <div className="c-storybook-example-footer">
-      {iframe}
       <ExampleFooter
-        html={htmlCode}
-        highlightedJsx={reactCode}
         sourceLink={
           <Button
             href={makeStorybookUrl(storyId, theme)}
             rel="noreferrer"
             variation="ghost"
             size="small"
           >
-            Open in Storybook
+            Open code in Storybook
```

Comment:
Open to ideas for what this link should say. I went with "code" to illustrate that's where you'll find the code docs, but I don't know if that's obvious or not.

Reply:
I agree that we need _something_ on the component page to point developers to Storybook, but the "Open code" part is a little misleading because you can't get to the example code from the story page, only from the doc page in Storybook. It would be nice if there was a way to "show code" on an individual story.

Reply:
`Storybook example / documentation`, `View component documentation`, `View interactive component documentation` ?